src/main/java/org/opensearch/performanceanalyzer/collectors/AdmissionControlMetricsCollector.java

```diff
@@ -40,9 +40,12 @@ public class AdmissionControlMetricsCollector extends PerformanceAnalyzerMetrics
     private static final String ADMISSION_CONTROL_SERVICE =
             "com.sonian.opensearch.http.jetty.throttling.JettyAdmissionControlService";
 
+    private final ClassLoader admissionControlClassLoaer;
+
     public AdmissionControlMetricsCollector() {
         super(sTimeInterval, "AdmissionControlMetricsCollector");
         this.value = new StringBuilder();
+        this.admissionControlClassLoaer = this.getClass().getClassLoader().getParent();
```

Comment:
Will `this.getClass().getClassLoader()` ever have Null value ? As I understand this should return the ApplicationClassLoader value ? Can you confirm.